scipy/stats/_multivariate.py

```diff
@@ -4693,13 +4693,14 @@ def rvs(self, m, n, size=None, random_state=None):
         ----------
         %(_doc_default_callparams)s
         size : integer or iterable of integers, optional
-            Number of samples to draw (default 1).
+            Number of samples to draw (default ``None``).
```

Comment:
```suggestion
            Number samples to draw. Default is ``None``, which...
```
"None" sounds like "no samples". I think it would be helpful to clarify here - not just below.
Is it enough to say "number of samples" or do we need to say something about the shape? How does broadcasting work?

Reply:
I have pushed a change to match NumPy. Though, if we want to be more clear, we would probably need to document the broadcasting behaviour. I don't know how exactly to proceed on that. How does this sound:

```none
        size : integer or iterable of integers, optional
            Number of samples to draw. Default is ``None``, in which case a
            single variate is returned as an array with shape ``np.broadcast(m, n).shape``.
```

Reply:
I don't think that's true. 
```
>>> multivariate_hypergeom.rvs([30, 40], [10, 2])
array([[4, 6],
       [2, 0]])
>>> np.broadcast([30, 40], [10, 2]).shape
(2,)
```
If CI passes, I'll merge this as-is given that it is an improvement, if that's OK? It seems to me you wanted to fix a mistake with this, whereas a complete description of the return shape is a bit beyond the original scope.

Reply:
> I don't think that's true

Ah, yes. My bad! Should be ``np.broadcast(m, n[None, ...]).shape`` but it's fine if it isn't included in the docs.

> If CI passes, I'll merge this as-is given that it is an improvement, if that's OK?

Yes, I am OK with the changes.